modell/src/main/kotlin/no/nav/tilbakekreving/behandling/saksbehandling/Foreldelsesteg.kt

```diff
@@ -23,6 +23,11 @@ class Foreldelsesteg(
 
     override fun erFullstendig(): Boolean = vurdertePerioder.all { it.vurdering != Vurdering.IkkeVurdert }
 
+    override fun nullstill() {
+        val nullstill = opprett(kravgrunnlag)
+        this.vurdertePerioder = nullstill.vurdertePerioder
```

Comment:
Syntes egentlig ikke dette blir helt riktig måte å løse det på, men ser at det blir kjipt å implementere den samme logikken to steder 🤔